draft-ietf-mls-architecture.md

```diff
@@ -293,7 +293,7 @@ The Authentication Service (AS) has to provide three functionalities:
 3. Enable a group member to verify that a credential represents the same client
    as another credential
 
-A member with a valid credential authenticates its MLS messages by signing them
+A member with a valid credential authenticates it's MLS messages by signing them
```

Comment:
```suggestion
A member with a valid credential authenticates its MLS messages by signing them
```

[This is incorrect.](https://www.merriam-webster.com/words-at-play/when-to-use-its-vs-its)  The MLS messages belong to the member, so we want the possessive here, not the contraction.